aea/cli/utils/constants.py

```diff
@@ -50,7 +57,27 @@
     PACKAGES,
     "aea",
 ]
-
+DEFAULT_CLI_CONFIG = {
```

Comment:
Let's open an issue to eventually also impose a json validation on this, like we do for the yaml files.

Reply:
#124